.github/actions/release/Makefile

```diff
@@ -3,8 +3,8 @@ BRANCH ?= $(shell git rev-parse --abbrev-ref HEAD)
 
 VERSION = $(shell git rev-parse --short=7 HEAD)
 IMAGE_NAME = spark-operator
-SPARK_BASE_IMAGE = gcr.io/ocean-spark/spark:platform-3.2-gen19
-TAG = $(VERSION)-gen19
+SPARK_BASE_IMAGE = gcr.io/ocean-spark/spark:platform-3.3-gen20
```

Comment:
We don't have a gen20 image of spark 3.2 .. so I updated to to 3.3. Not 100% sure this is safe to upgrade or we should just address the vulnerabilities in the base image ? wdyt @ImpSy @thorsteinnth ?
- The gcr https://console.cloud.google.com/gcr/images/ocean-spark/global/spark